scripts/build_cutout.py

```diff
@@ -92,28 +88,38 @@
 """
 
 import logging
+import atlite
+import geopandas as gpd
+import pandas as pd
 from _helpers import configure_logging
 
-import os
-import atlite
+
+logger = logging.getLogger(__name__)
 
 logger = logging.getLogger(__name__)
 
 if __name__ == "__main__":
     if 'snakemake' not in globals():
         from _helpers import mock_snakemake
-        snakemake = mock_snakemake('build_cutout', cutout='europe-2013-era5')
+        snakemake = mock_snakemake('build_cutout', cutout='base')
     configure_logging(snakemake)
 
     cutout_params = snakemake.config['atlite']['cutouts'][snakemake.wildcards.cutout]
-    for p in ('xs', 'ys', 'years', 'months'):
-        if p in cutout_params:
-            cutout_params[p] = slice(*cutout_params[p])
 
-    cutout = atlite.Cutout(snakemake.wildcards.cutout,
-                        cutout_dir=os.path.dirname(snakemake.output[0]),
-                        **cutout_params)
+    snapshots = pd.date_range(freq='h', **snakemake.config['snapshots'])
+    time = [snapshots[0], snapshots[-1]]
```

Comment:
I am again assuming the `time` if it is not specified in `config.yaml` for the cutout.
The script should rather fail than trying to fix it.